pkg/drm/drm.go

```diff
@@ -0,0 +1,13 @@
+package drm
+
+// TODO DRM class
+const (
+	SCHEME_LCP = "http://readium.org/2014/01/lcp"
+)
+
+type DRMLicense interface {
```

Comment:
The `DRMLicense` interface is actually deprecated, now everything goes through the `ContentProtection` API.

Although I realised that the type is not marked as deprecated in the Kotlin toolkit, even though we are not using it anymore.

Reply:
Commenting it out for now, added comment to revisit when LCP/content protection is implemented in the go toolkit